taccsite_cms/templates/guides/rsync_data_transfer.html

```diff
@@ -0,0 +1,510 @@
+            </li>
+          </ul>
+        </li>
+        <li>
+          <a href="#local_data_path">Determining the Path to Your Transfer Directory</a>
+        </li>
+        <li>
+          <a href="#scp_transfer">Transfer Data with <code>scp</code></a>
+        </li>
+        <li>
+          <a href="#sftp_transfer">Transfer Data with <code>sftp</code></a>
+        </li>
+        <li>
+          <a href="#rsync_transfer">Transfer Data with <code>rsync</code></a>
+        </li>
+        <li>
+          <a href="#cyberduck_transfer">Transfer Data with Cyberduck</a>
+          <ul>
+            <li>
+              <a href="#cy_windows">For Windows</a>
+              <ul>
+                <li>
+                  <a href="#figure2">Figure 2. Windows Cyberduck and “Open Connection” setup screen</a>
+                </li>
+                <li>
+                  <a href="#figure3">Figure 3. Windows “Open Connection” setup screen</a>
+                </li>
+              </ul>
+            </li>
+            <li>
+              <a href="#cy_mac">For Mac</a>
+              <ul>
+                <li>
+                  <a href="#figure4">Figure 4. macOS “New Bookmark” setup screen</a>
+                </li>
+              </ul>
+            </li>
+          </ul>
+        </li>
+        <li>
+          <a href="#references">References</a>
+        </li>
+      </ul>
+
+      <h3>Data Transfer and Management Guide</h3>
+
+      <p>
+        Transferring Portal data from your local machine to
+        <a href="#storage">one of TACC's remote storage systems</a> can be accomplished using two methods: command line tools (<code>scp</code>, <code>sftp</code>,
+        <code>rsync</code>) and graphical tools (Cyberduck).
+      </p>
+
+      <h4 id="table1">Data Transfer Methods</h4>
+      <table style="max-width: 500px">
+        <tbody>
+          <tr>
+            <th>Usage Mode</th>
+            <th>Transfer Method</th>
+          </tr>
+          <tr>
+            <td>Command Line Tool</td>
+            <td>
+              <a target="_blank" href="https://man7.org/linux/man-pages/man1/scp.1.html"><code>scp</code></a>
+            </td>
+          </tr>
+          <tr>
+            <td>Command Line Tool</td>
+            <td>
+              <a target="_blank" href="https://man7.org/linux/man-pages/man1/sftp.1.html"><code>sftp</code></a>
+            </td>
+          </tr>
+          <tr>
+            <td>Command Line Tool</td>
+            <td>
+              <a target="_blank" href="https://man7.org/linux/man-pages/man1/rsync.1.html"><code>rsync</code></a>
+            </td>
+          </tr>
+          <tr>
+            <td>Graphical Tool</td>
+            <td>
+              <a target="_blank" href="https://cyberduck.io/">Cyberduck</a>
+            </td>
+          </tr>
+        </tbody>
+      </table>
+      <h4 id="storage">What is TACC's Storage Server?</h4>
+
+      <p>
```

Comment:
A TACC storage system is a logically defined resource designed to provide data storage and management capabilities to TACC portal users through the portal interface. Each portal is accessible over a URL path (e.g., `sub.domain.tacc.utexas.edu`, `domain.org`) which we will subsequently refer to as "host".

Storage systems can be configured for both normal and protected data (e.g., HIPAA) in a secured location, depending on the project requirements established for the portal. This location is exposed as a path to a directory on the secure system (e.g., `/secure-server-root/projects/portal/directory_name`) which this document will subsequently refer to as `/transfer/directory/path`. 

Storage systems are to be used exclusively for transferring and accessing data through the portal.